RELEASE_NOTES.md

```diff
@@ -15,6 +15,7 @@ Release 0.2.0 coincides with MESSAGEix release 1.2.0.
 
 ## All changes
 
+- [#165](https://github.com/iiasa/ixmp/pull/165): Add support for handling geodata.
```

Comment:
This should be under "Next Release", not under "v0.2.0 / All changes" (since 0.2.0 was already released).